lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/editions/workflow_controller.rb

```diff
@@ -47,6 +47,11 @@ def update
     end
   end
 
+  def context
```

Comment:
this is a good idea!

Reply:
I do wonder whether we should be a bit better at having microcopy like this in translation files. It's always one of those things that you start off thinking it's not worth doing, then as a project gets bigger you wish you'd have done sooner!